test/Azure.Functions.Cli.Tests/E2E/Helpers/TestingTraits.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Functions.Cli.Tests.E2E.Helpers
+{
+    internal static class TestingTraits
+    {
+        internal class TestCategory
```

Comment:
I don't think we need these extra internal classes; I also have a change that introduces test traits here:

https://github.com/Azure/azure-functions-core-tools/blob/liliankasem/test-collections/test/Azure.Functions.Cli.Tests/Utilities/TestTraits.cs

So I wonder if we can follow a similar pattern so it would be easy to combine these two later when my PR is open

Reply:
changed it to follow your pattern!